lib/awesome_spawn.rb

```diff
@@ -164,7 +164,7 @@ def launch(env, command, spawn_options)
   def parse_command_options(command, options)
     options = options.dup
     params  = options.delete(:params)
-    env = options.delete(:env) || {}
+    env = (options.delete(:env) || {}).map{ |n,v| [n, v.to_s] }.to_h
```

Comment:
Probably should .to_s the key as well, since things like symbols aren't allowed as keys:

```
spawn({:ABC => "asdf"}, "echo \"'$ABC'\"")
Traceback (most recent call last):
	 1: from (pry):8:in `__pry__'
(pry):8:in `spawn': no implicit conversion of Symbol into String (TypeError)
```

Reply:
added a spec that exercised your test case as well.